packages/cli/package.json

```diff
@@ -31,136 +31,136 @@
-    "@rollup/plugin-commonjs": "25.0.4",
-    "@rollup/plugin-json": "6.0.0",
-    "@rollup/plugin-node-resolve": "15.2.1",
-    "@rollup/plugin-yaml": "4.0.0",
-    "@spotify/eslint-config-base": "15.0.0",
-    "@spotify/eslint-config-react": "15.0.0",
-    "@spotify/eslint-config-typescript": "15.0.0",
-    "@sucrase/webpack-loader": "2.0.0",
-    "@svgr/core": "8.1.0",
-    "@svgr/plugin-jsx": "8.1.0",
-    "@svgr/plugin-svgo": "6.5.1",
-    "@svgr/rollup": "8.1.0",
-    "@svgr/webpack": "6.5.1",
-    "@swc/core": "1.3.46",
-    "@swc/helpers": "0.5.0",
-    "@swc/jest": "0.2.22",
-    "@types/jest": "29.5.5",
-    "@types/webpack-env": "1.15.2",
-    "@typescript-eslint/eslint-plugin": "5.62.0",
-    "@typescript-eslint/parser": "5.62.0",
-    "@yarnpkg/lockfile": "1.1.0",
-    "@yarnpkg/parsers": "3.0.0-rc.4",
-    "bfj": "7.0.2",
-    "buffer": "6.0.3",
-    "chalk": "4.0.0",
-    "chokidar": "3.3.1",
-    "commander": "9.1.0",
-    "cross-fetch": "3.1.5",
-    "cross-spawn": "7.0.3",
-    "css-loader": "6.5.1",
-    "diff": "5.0.0",
-    "esbuild": "0.19.0",
-    "esbuild-loader": "2.18.0",
-    "eslint": "8.49.0",
-    "eslint-config-prettier": "8.10.0",
-    "eslint-formatter-friendly": "7.0.0",
-    "eslint-plugin-deprecation": "1.5.0",
-    "eslint-plugin-import": "2.28.1",
-    "eslint-plugin-jest": "27.4.0",
-    "eslint-plugin-jsx-a11y": "6.7.1",
-    "eslint-plugin-react": "7.33.2",
-    "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-webpack-plugin": "3.2.0",
-    "express": "4.17.1",
-    "fork-ts-checker-webpack-plugin": "7.0.0-alpha.8",
-    "fs-extra": "10.1.0",
-    "git-url-parse": "13.0.0",
-    "glob": "7.1.7",
-    "global-agent": "3.0.0",
-    "handlebars": "4.7.7",
-    "html-webpack-plugin": "5.3.1",
-    "inquirer": "8.2.0",
-    "jest": "29.7.0",
-    "jest-css-modules": "2.1.0",
-    "jest-environment-jsdom": "29.7.0",
-    "jest-runtime": "29.7.0",
-    "json-schema": "0.4.0",
-    "lodash": "4.17.21",
-    "mini-css-extract-plugin": "2.4.2",
-    "minimatch": "5.1.1",
-    "node-fetch": "2.6.7",
-    "node-libs-browser": "2.2.1",
-    "npm-packlist": "5.0.0",
-    "ora": "5.3.0",
-    "postcss": "8.2.13",
-    "process": "0.11.10",
-    "react-dev-utils": "12.0.0-next.60",
-    "react-refresh": "0.14.0",
-    "recursive-readdir": "2.2.2",
-    "replace-in-file": "6.0.0",
-    "rollup": "2.78.0",
-    "rollup-plugin-dts": "4.0.1",
-    "rollup-plugin-esbuild": "4.7.2",
-    "rollup-plugin-postcss": "4.0.0",
-    "rollup-pluginutils": "2.8.2",
-    "run-script-webpack-plugin": "0.2.0",
-    "semver": "7.5.2",
-    "style-loader": "3.3.1",
-    "sucrase": "3.20.2",
-    "swc-loader": "0.2.3",
-    "tar": "6.1.12",
-    "terser-webpack-plugin": "5.1.3",
-    "util": "0.12.3",
-    "webpack": "5.76.0",
-    "webpack-dev-server": "4.7.3",
-    "webpack-node-externals": "3.0.0",
-    "yaml": "2.2.2",
-    "yml-loader": "2.1.0",
-    "yn": "4.0.0",
-    "zod": "3.21.4"
+    "@backstage/catalog-model": "^1.4.3",
```

Comment:
Yeah, I'm trying to relax pinned versions in most of the packages - where it makes sense since the pinning of `dependencies` is really useful when you have a JS application that you actually deploy. We don't do that, so we don't necessarily need that. After all, Yarn 1 is really not the best dependency resolution engine, so fixed versions in many places are really causing duplications and unwanted version mismatches.